styletesla.css

```diff
@@ -1,97 +1,640 @@
 	display: flex;
 	justify-content: space-between;
-
-}
+    }
+.advantage__title {
+	font: 70px/85px montserratsemibold;
+	padding-left: 77px;
+	color: #E51937;
+    }
 .sidebar {
-	height: 100vh;
-	width: 30%;
+	height: 100%;
+	width: 28%;
 	right: 0;
 	background:#E51937;
-
+	padding-left: 50px;
+	color: white;
+	}
+.sidebar__title {
+	font: 30px/35px montserratsemibold;
+	padding-top: 3vh;
+	margin: 0;
+	margin-bottom: 5px;
+    }
+.sidebar__discription {
+	font: 15px/18px montserratmedium;
+    }
+.list {
+	list-style: none;
+	padding-left: 0;
+    }
+.list p {
+	margin: 15px;
+    }
+.icon_dujm, .icon_po, .icon_litr {
+	display: flex;
+	align-items: center;
+    }
+.icon_screen {
+	width: 36px;
+	height: 53px;
+    }
+.icon_wifi {
+	width: 44px;
+	height: 34px;
+    }
+.icon_bag {
+	width: 36px;
+	height: 52px;
+    }
+.po {
+	font: 20px/18px montserratsemibold;
+}
+.list {
+	margin-bottom: 170px ;
+}
+.button {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 70px;
+}
+.button__more {
+	display: flex;
+	align-items: center;
+}
+.more__sign {
+	width: 22px;
+	height: 22px;
+	margin-right: 5px;
+}
+.know { 
+	font: 13px/16px montserratbold;
+	cursor: pointer;
+}
+.button__order {
+    border: 2px solid white;
+	color: white;
+	background: #E51937;
+	font: 13px/16px montserratbold;
+	text-transform: uppercase;
+	margin-right: 30px;
+}
+/*планшет*/
+@media (max-width: 1270px) {
+	.sidebar {
+	width: 35%;
+	}
+}
+@media (max-width: 1165px) {
+.advantage {
+	height: 900px;
+	width: 100%;
+	background: url('img/rulj-planshet.png') top no-repeat; 
+	background-size: contain;
+	display: flex;
+	flex-direction: column;
 }
+.advantage__title {
+	padding-left: 54%;
+	margin-top: 3%;
+	}
+.sidebar {
+	width: 100%;
+	height: 51%;
+	padding-left: 0;
+	display: grid;
+	grid-template-columns: 53% 47%;
+	grid-template-rows: 80% 20%;
+    }
+.sidebar__title, .sidebar__discription {
+	grid-column: 1/2;
+	grid-row: 1/2;
+	padding-left: 50px;
+    }
+.sidebar__discription {
+	padding-top: 90px;
+}
+.list{
+	grid-column: 2/3;
+	padding-top: 35px;
+	margin-bottom: 0;
+}
+.button {
+	grid-column: 1/3;
+	display: flex;
+	justify-content: center;
+   }
+.button__more {
+	padding-right: 120px;
+}
+}
+@media (max-width: 790px) {
+.sidebar {
+	width: 100%;
+	height: 55%;
+    }
 .sidebar__title, .sidebar__discription {
-	color:white;
+	padding-left: 25px;
+	}
+.list {
+	padding-left: 40px;
+	}
 }
+@media (max-width: 670px) {
+.advantage__title, .list, .button {
+		display: none;
+	}
+.advantage {
+	height: 580px;
+	width: 100%;
+	/*можно почистить?*/
+	background: url('img/rulj-planshet.png') top no-repeat; 
+	background-size: contain;
+	display: flex;
+	flex-direction: row;
+	align-items: flex-end;
+}
+.sidebar {
+	display: flex;
+	flex-direction: column;
+	height: 58%;
+	}
+.sidebar__title, .sidebar__discription {
+	margin: 0 auto;
+	}
 .sidebar__title {
-	padding-top: 51px;
-	margin: 0;
+	padding: 20px 0;
+}
+.sidebar__discription {
+	padding: 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5512124)